selfdrive/ui/translations/main_nl.ts

```diff
@@ -0,0 +1,1305 @@
+    <name>DriveStats</name>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="37"/>
+        <source>Drives</source>
+        <translation>Ritten</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="39"/>
+        <source>Hours</source>
+        <translation>Uren</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="44"/>
+        <source>ALL TIME</source>
+        <translation>TOTAAL</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="46"/>
+        <source>PAST WEEK</source>
+        <translation>AFGELOPEN WEEK</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.h" line="15"/>
+        <source>KM</source>
+        <translation>Km</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.h" line="15"/>
+        <source>Miles</source>
+        <translation>Mijl</translation>
+    </message>
+</context>
+<context>
+    <name>DriverViewScene</name>
+    <message>
+        <location filename="../qt/offroad/driverview.cc" line="55"/>
+        <source>camera starting</source>
+        <translation>Camera wordt gestart</translation>
+    </message>
+</context>
+<context>
+    <name>InputDialog</name>
+    <message>
+        <location filename="../qt/widgets/input.cc" line="71"/>
+        <source>Cancel</source>
+        <translation>Annuleren</translation>
+    </message>
+    <message numerus="yes">
+        <location filename="../qt/widgets/input.cc" line="168"/>
+        <source>Need at least %n character(s)!</source>
+        <translation>
+            <numerusform>Heeft minstens %n karakter nodig!</numerusform>
+            <numerusform>Heeft minstens %n karakters nodig!</numerusform>
+        </translation>
+    </message>
+</context>
+<context>
+    <name>Installer</name>
+    <message>
+        <location filename="../installer/installer.cc" line="56"/>
+        <source>Installing...</source>
+        <translation>Installeren...</translation>
+    </message>
+    <message>
+        <location filename="../installer/installer.cc" line="144"/>
+        <source>Receiving objects: </source>
+        <translation>Objecten ontvangen: </translation>
+    </message>
+    <message>
+        <location filename="../installer/installer.cc" line="145"/>
+        <source>Resolving deltas: </source>
+        <translation>Deltas verwerken: </translation>
+    </message>
+    <message>
+        <location filename="../installer/installer.cc" line="146"/>
+        <source>Updating files: </source>
+        <translation>Bestanden bijwerken: </translation>
+    </message>
+</context>
+<context>
+    <name>MapETA</name>
+    <message>
+        <location filename="../qt/maps/map.cc" line="622"/>
+        <source>eta</source>
+        <translation>verwachte aankomsttijd</translation>
```

Comment:
This needs to be an abbreviation to fit in the bottom bar of the navigation. Let's just keep "eta".
```suggestion
        <translation>eta</translation>
```